src/encode_task_idr.py

```diff
@@ -147,13 +147,14 @@ def main():
         args.peak1, args.peak2, args.peak_pooled, args.peak_type,
         args.idr_thresh, args.idr_rank, args.out_dir)
 
+    log.info('Checking if output is empty...')
+    assert_file_not_empty(idr_peak, help=
+        'No IDR peaks found. IDR threshold might be too stringent?')
```

Comment:
The more usual explanation for empty IDR peak set is mis-matched replicates (sample swap), one totally failed replicate, or just a bad experiment. Maybe say "IDR threshold might be too stringent, or replicates have very poor concordance."